portfolio/src/main/webapp/index.html

```diff
@@ -81,6 +81,25 @@ <h1>Adam's Personal Portfolio</h1>
           <li>Friends and family also make an appearance!</li>
         </ul>  
       </p>
+
+      <div id="comment">
+        <h2>Leave a Comment</h2>
+        <form action="/data" method="POST">
+
+        
+        <label for="user-name">Your name:</label></br>
+        <input type="text" name="user-name">
+        </br></br>
+
+        //change format for comment box
+        <label for="user-comment">Your comment:</label></br>
+        <input type="text" name"user-comment">
+        </form>
+
+      </div>
+
+
+
       <button onclick="getData()">Get the Data URL</button>
```

Comment:
I'm sorry, I missed this in the previous CL! But why did you name it "Get the Data URL"? That won't make much sense to a user of your website. Consider a different name?

Reply:
at the time it was mostly just to test that the button worked. But you are right I will change it!